### PR TITLE
Proposal:  Permissions Documentation

### PR DESCRIPTION
**Pull request by Joshua-Anderson**
This is a continuation of #4150, adding in-depth docs about permissions, including api docs.

### CODE REVIEW

File: docs/customizing_deis/controller_settings.rst [hunk 1]
```diff
@@ -134,6 +135,37 @@ This will set the registration mode to admin_only.
 
     $ deisctl config controller set registrationMode="admin_only"
 
+.. _configure_default_permissions:
+
+Configuring Default Permissions
+-------------------------------
+These settings can be used to configure the default permissions for users.
+These settings are retroactive and effect all current and new users.
```
Comment (krancour) [inline docs/customizing_deis/controller_settings.rst]: Should be "affect".

File: docs/using_deis/permissions.rst [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+-------------------
+
+By default, users have the following permissions: ``app``, ``certs``.
+When invited to an app, users have the ``push``, ``config``, ``domains``, ``scale`` permissions by default.
+
+Changing the Default Permissions
+--------------------------------
+
+Default apps permissions are stored in etcd and can be updated with ``deisctl``.
+Updating default permissions is retroactive and effects current and users.
```
Comment (krancour) [inline docs/using_deis/permissions.rst]: Should be "affect".

File: docs/using_deis/permissions.rst [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+This does not effect users that have had a permission explicitly set.
```
Comment (krancour) [inline docs/using_deis/permissions.rst]: Should be "affect".

File: docs/using_deis/permissions.rst [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+the default permission to allow users to create apps, that user still couldn't create
+apps.
+
+See :ref:`configure_default_permissions` to learn how to configure default permissions.
+
+Setting Permissions
+-------------------
+
+To grant a permission to a user, you can run ``deis perms:create <username> --<permission>``
+Fr example, to grant user foo permission to create an app ``deis perms:create foo --app``
```
Comment (krancour) [inline docs/using_deis/permissions.rst]: > Fr example

"For example"

**Comment by Joshua-Anderson** [comment]
Thanks for the catches @krancour. I also moved the proposal to the v1.6 api now that that is out.

**Comment by krancour** [comment]
Obviously, I think we need to rethink the permissions model overall when a major release eventually permits us to do so, but in terms of grafting a more finely-grained permissions model onto the existing platform without affecting backwards-compatibility, I think this proposal is very nice.

Proposal LGTM

**Comment by carmstrong** [comment]
Reassigning to @mboersma to take over for this release, since @Joshua-Anderson leaves us on Friday... :cry:

**Comment by kalbasit** [comment]
Accidentally closed the PR.

**Comment by carmstrong** [comment]
@Joshua-Anderson Needs a rebase.

**Comment by Joshua-Anderson** [comment]
@carmstrong I'm not sure this is ever going to be merged.... the docs about changing the default permissions are in my default permissions pr #4265. I mostly left this open so that @mboersma can copy over the relevant documentation when he opens a per user permissions PR and forgot about it.

**Comment by Joshua-Anderson** [comment]
Closing this. I'll look at re-implementing in V2 in the future.

**Pull request closed without merging**